deploy/llm-image/templates/clusterversion.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ClusterVersion
+metadata:
+  name: llm-gpt2
+  labels:
+    {{- include "llm.labels" . | nindent 4 }}
+spec:
+  clusterDefinitionRef: llm
+  componentVersions:
+    - componentDefRef: llm
+      configSpecs:
+        - name: llm-ts-handler-config-gpt2
+          templateRef: llm-ts-handler-config-gpt2
+          volumeName: ts-handler-config
+          namespace: {{ .Release.Namespace }}
+      versionsContext:
+        initContainers:
+          - name: model
+            image: apecloud/gpt2:0.0.1
```

Comment:
better to setup the image registry with template `{{ .Values.image.registry | default "docker.io" }}`.